### ISSUE TITLE
Void label

### ISSUE THREAD

**Issue by saskra** [alicranck/coco2voc#10]
Thanks for this tool for converting, nice that it can do polygon masks too! However, I noticed two differences in the result compared to the original data:
1. the first would be just a cosmetic thing, and that is that the original images seem to have a color map/ LUT applied.
2. possibly more important would be that in the originals there is an additional border around the objects in another color, see below. Does this probably have an impact on the training and could this be added to your feature?

![index](https://user-images.githubusercontent.com/12963485/234325038-37311677-fb09-4869-a61b-7c38e0da921e.png)

**Comment by alicranck** [comment]
Hi @saskra, Thanks for raising the issue. About the color map, I intentionally kept the mask values as the coco object ids so that they can be used in training as is, although it's less convinient to view. I'm not sure if it's a good idea to change at this point.

As for the border I will look into it. Feel free to link or suggest edits as well.

Thanks!

**Comment by saskra** [comment]
Thanks for the answer! Adding a Color Lookup Table or indexed colors should not change the values of the individual pixels per se. I did this once with ImageJ/ Fiji and the Glasbey LUT on one of the instance label images I got with your script, see below. (By the way, in my case there is only one class, and it doesn't belong to the original COCO objects, so I wouldn't really care about the ID). I've also achieved this effect with Python, but on TIFF images, for PNGs I'd have to figure it out first. But as I said, the other point seems even more relevant to me.

![example_glasybeyLUT](https://user-images.githubusercontent.com/12963485/235112065-7b86ac77-3112-40e9-a873-b9ba12fef94d.png)
